awx/main/dispatch/worker/task.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import importlib
+import inspect
+import sys
+import traceback
+
+import six
+
+from awx.main.tasks import dispatch_startup, inform_cluster_of_shutdown
+
+from .base import BaseWorker
+
+logger = logging.getLogger('awx.main.dispatch')
+
+
+class TaskWorker(BaseWorker):
```

Comment:
This worker implementation takes messages published via `.apply_async()`, deserializes them, and runs native Python code.  This does what celery workers did in 3.3.0.